### DISCUSSION TITLE
Dolphin died after update

### DISCUSSION THREAD

**Discussion by 4thehalibit** [prasanthrangan/hyprdots#1147] [Q&A]
After i updated, Dolphin seems to have become corrupt
![image](https://github.com/prasanthrangan/hyprdots/assets/12929552/9fcda974-9743-4fff-a3cd-d77623df2830)

**Comment by kRHYME7** [comment]
System update?  See this https://github.com/prasanthrangan/hyprdots/issues/1030

This is a dupe
